Author: Unrud <unrud@outlook.com>
Description: fix a bug crashing the game when applying new settings
Last-Update: 2021-04-05
--- a/src/FretsOnFire.py
+++ b/src/FretsOnFire.py
@@ -86,6 +86,7 @@
         pass
     except KeyboardInterrupt:
         pass
+    engine.audio.close()
 
     if engine.restartRequested:
       Log.notice("Restarting.")
--- a/src/GameEngine.py
+++ b/src/GameEngine.py
@@ -246,7 +246,6 @@
       self.quit()
     
   def quit(self):
-    self.audio.close()
     Engine.quit(self)
 
   def resizeScreen(self, width, height):
